UT4MasterServer.Web/src/routes.ts

```diff
@@ -4,7 +4,8 @@ import { adminGuard, privateGuard, publicGuard } from './route-guards';
 export const routes: RouteRecordRaw[] = [
   // Secure Pages
   {
-    path: `/Stats/:accountId?`,
+    name: 'Stats',
+    path: `/Stats/:window?/:accountId?`,
```

Comment:
I think this should be swapped, so that just having account id in url is possible.
```suggestion
    path: `/Stats/:accountId?/:window?`,
```